packages/Skipper/SkipCriteriaResolver/SkippedPathsResolver.php

```diff
@@ -40,12 +40,12 @@ public function resolve(): array
                 continue;
             }
 
-            if (file_exists($value)) {
+            if (\str_contains((string) $value, '*')) {
```

Comment:
switched the conditions so we do the cheaper string check before the IO bound one